logging/README.md

```diff
@@ -248,6 +251,8 @@ which you cloned the repository and issue this command:
 
 The script creates the namespace into which the components are deployed. By default, the components are deployed into the namespace `logging`.
 
+**Note:* If you are deploying SAS Viya Logging on OpenShift, you must follow the deployment process documented in [SAS Viya Logging on OpenShift](/openshift/openshift.md).**
```

Comment:
Link is broken - should be `/logging/openshift/openshift.md` or `openshift/openshift.md`

Reply:
Moved the logging/openshift/openshift.md to logging/OpenShift.md to be consistent with monitoring; fixed references in top-level README.md.